haystack/document_store/elasticsearch.py

```diff
@@ -492,6 +492,7 @@ def _get_all_documents_in_index(
         index: str,
         filters: Optional[Dict[str, List[str]]] = None,
         batch_size: int = 10_000,
+        filter_documents_without_embedding: bool = False,
```

Comment:
How about renaming this to `only_documents_without_embedding`? 
With the current naming it is not clear to me if `True` means I will get only those docs or everything except them.